Author: gregor herrmann <gregoa@debian.org>
Description: turn off LCSS reporting in newer Test::LongString
 require Test::LongString 0.13
Bug: http://rt.cpan.org/Public/Bug/Display.html?id=53637
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=566585
Forwarded: http://rt.cpan.org/Public/Bug/Display.html?id=53637
Origin: vendor

--- a/Mechanize.pm
+++ b/Mechanize.pm
@@ -65,7 +65,7 @@
 =cut
 
 use WWW::Mechanize ();
-use Test::LongString;
+use Test::LongString lcss => 0;
 use Test::Builder ();
 use Carp ();
 use Carp::Assert::More;
--- a/META.yml
+++ b/META.yml
@@ -12,7 +12,7 @@
     HTTP::Server::Simple:          0.35
     HTTP::Server::Simple::CGI:     0
     Test::Builder::Tester:         1.09
-    Test::LongString:              0.07
+    Test::LongString:              0.13
     Test::More:                    0
     URI::file:                     0
     WWW::Mechanize:                1.24
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -13,7 +13,7 @@
         'HTTP::Server::Simple'      => '0.35',
         'HTTP::Server::Simple::CGI' => 0,
         'Test::Builder::Tester'     => '1.09',
-        'Test::LongString'          => '0.07',
+        'Test::LongString'          => '0.13',
         'Test::More'                => 0,
         'URI::file'                 => 0,
         'WWW::Mechanize'            => '1.24',
